core/aggregation.go

```diff
@@ -99,240 +85,37 @@ type SignatureAggregator interface {
-					"accepted for batch %x. Uncollected signature count: %d", message, remainingReplies)
-			contextExpired = true
-		}
-		if contextExpired {
-			break
-		}
-
-		if seen := signerMap[r.Operator]; seen {
-			a.Logger.Warn("duplicate signature received", "operatorID", r.Operator.Hex())
-			continue
-		}
-
-		operatorIDHex := r.Operator.Hex()
-		operatorAddr, ok := a.OperatorAddresses.Get(r.Operator)
-		if !ok && a.Transactor != nil {
-			operatorAddr, err = a.Transactor.OperatorIDToAddress(ctx, r.Operator)
-			if err != nil {
-				a.Logger.Warn("failed to get operator address from registry", "operatorID", operatorIDHex)
-				operatorAddr = gethcommon.Address{}
-			} else {
-				a.OperatorAddresses.Add(r.Operator, operatorAddr)
-			}
-		} else if !ok {
-			operatorAddr = gethcommon.Address{}
-		}
-
-		socket := ""
-		if op, ok := state.IndexedOperators[r.Operator]; ok {
-			socket = op.Socket
-		}
-		batchHeaderHashHex := hex.EncodeToString(r.BatchHeaderHash[:])
-		if r.Err != nil {
-			a.Logger.Warn("error returned from messageChan",
-				"operatorID", operatorIDHex,
-				"operatorAddress", operatorAddr,
-				"socket", socket,
-				"batchHeaderHash", batchHeaderHashHex,
-				"attestationLatencyMs", r.AttestationLatencyMs,
-				"err", r.Err)
-			continue
-		}
-
-		op, found := state.IndexedOperators[r.Operator]
-		if !found {
-			a.Logger.Error("Operator not found in state",
-				"operatorID", operatorIDHex,
-				"operatorAddress", operatorAddr,
-				"socket", socket)
-			continue
-		}
-
-		// Verify Signature
-		sig := r.Signature
-		ok = sig.Verify(op.PubkeyG2, message)
-		if !ok {
-			a.Logger.Error("signature is not valid",
-				"operatorID", operatorIDHex,
-				"operatorAddress", operatorAddr,
-				"socket", socket,
-				"pubkey", hexutil.Encode(op.PubkeyG2.Serialize()))
-			continue
-		}
-
-		operatorQuorums := make([]uint8, 0, len(quorumIDs))
-		for _, quorumID := range quorumIDs {
-			// Get stake amounts for operator
-			ops := state.Operators[quorumID]
-			opInfo, ok := ops[r.Operator]
-			// If operator is not in quorum, skip
-			if !ok {
-				continue
-			}
-			operatorQuorums = append(operatorQuorums, quorumID)
-
-			signerMap[r.Operator] = true
-
-			// Add to stake signed
-			stakeSigned[quorumID].Add(stakeSigned[quorumID], opInfo.Stake)
-
-			// Add to agg signature
-			if aggSigs[quorumID] == nil {
-				aggSigs[quorumID] = &Signature{sig.Clone()}
-				aggPubKeys[quorumID] = op.PubkeyG2.Clone()
-			} else {
-				aggSigs[quorumID].Add(sig.G1Point)
-				aggPubKeys[quorumID].Add(op.PubkeyG2)
-			}
-		}
-		a.Logger.Info("received signature from operator",
-			"operatorID", operatorIDHex,
-			"operatorAddress", operatorAddr,
-			"socket", socket,
-			"quorumIDs", fmt.Sprint(operatorQuorums),
-			"batchHeaderHash", batchHeaderHashHex,
-			"attestationLatencyMs", r.AttestationLatencyMs)
+	// tick interval is very long, since we don't actually need periodic attestation updates in this case. we only
```

Comment:
what does "this case" mean? V1?

Reply:
It occurred to me that it's easily doable in this PR to leave V1 functionality entirely unchanged, and to just reimplement with the necessary changes for V2.

I reverted changes that affect V1 functionality in 67d8e3c5aac5adb255ed65d42eb5364782e4408b, so the comment you're referring to is now gone entirely